### PR TITLE
explain the context attribute

### PR DESCRIPTION
**Pull request by jamesscottbrown**

### CODE REVIEW

**Comment by rpgoldman** [comment]
I don't think that description for context can be correct.  For example, asking for a set of instances, I get

```
{
  "@id": "https://sift.net/container-ontology/container-ontology#Plate",
  "hasInstance": <....>,
  "@context": "https://owlery.phenoscape.org/json/context.jsonld"
}
```

I really don't know what that `@context` field is supposed to be.

**Comment by jamesscottbrown** [comment]
> I don't think that description for context can be correct.

Why not? I think it is consistent with the response.

If I fetch the URL `https://owlery.phenoscape.org/json/context.jsonld`, the response is:

```json
{
  "@context": {
    "equivalentClass": {
      "@id": "http://www.w3.org/2002/07/owl#equivalentClass",
      "@type": "@id"
    },
    "subClassOf": {
      "@id": "http://www.w3.org/2000/01/rdf-schema#subClassOf",
      "@type": "@id"
    },
    "superClassOf": {
      "@reverse": "http://www.w3.org/2000/01/rdf-schema#subClassOf",
      "@type": "@id"
    },
    "type": "@type",
    "hasInstance": {
      "@reverse": "http://www.w3.org/1999/02/22-rdf-syntax-ns#type",
      "@type": "@id"
    },
    "isSatisfiable": {
      "@id": "http://purl.org/phenoscape/owlery.owl#isSatisfiable",
      "@type": "http://www.w3.org/2001/XMLSchema#boolean"
    },
    "value": {
      "@id": "http://www.w3.org/1999/02/22-rdf-syntax-ns#value"
    },
    "version": {
      "@id": "http://www.w3.org/2002/07/owl#versionIRI",
      "@type": "@id"
    },
    "imports": {
      "@id": "http://www.w3.org/2002/07/owl#imports",
      "@type": "@id"
    }
  }
}
```

Within this, the `hasInstance` attribute has the value:

```json
 {
      "@reverse": "http://www.w3.org/1999/02/22-rdf-syntax-ns#type",
      "@type": "@id"
}
```

Fetching `http://www.w3.org/1999/02/22-rdf-syntax-ns#type`, I see it is defined as:

```rdf
rdf:type a rdf:Property ;
	rdfs:isDefinedBy <http://www.w3.org/1999/02/22-rdf-syntax-ns#> ;
	rdfs:label "type" ;
	rdfs:comment "The subject is an instance of a class." ;
	rdfs:range rdfs:Class ;
	rdfs:domain rdfs:Resource .
```

So it seems that the `context.jsonld` file is defining `hasInstance` (in the context of the API's repsonse) to be the reverse of `rdf:type` (which is defined as "The subject is an instance of a class.").

In other words, this is sating that that x `hasInstance` y means that y is of class x. This seems to be consistent with the API results: the `hasInstance` attribute of the response contains a list of things which are instances of `container-ontology:Plate` - i.e., things which are of type `container-ontology:Plate`.

**Comment by rpgoldman** [comment]
OK, I think I see: `@context` is a JSON LD thing. The context it provides is something like a namespace import. It tells us how to interpret the use of the property `"hasInstance"` that appears in the query response without a namespace IRI.
